cluster/gce/trusty/node.yaml

```diff
@@ -262,6 +262,35 @@ end script
 MIME-Version: 1.0
 Content-Type: text/upstart-job; charset="us-ascii"
 Content-Transfer-Encoding: 7bit
+Content-Disposition: attachment; filename="kube-install-additional-packages.conf"
+
+#upstart-job
+
+description "Install additional packages used by kubernetes"
+
+start on stopped kube-docker
+
+script
+	# Socat and nsenter are used in port forwarding. They are not required for spin
+	# up a cluster. Therefore, we move the installation here to make it parallel to
+	# other steps to reduce the cluster creation time.
```

Comment:
I am not quite sure about the importance of socat and nsenter in k8s. My understanding is that they are for port forwarding. The situation is similar to kube-system containers. Before kube-system containers are all up, a cluster is non-functional. When kube-system containers are all up, this installation here is probably finished (manually checked in my experiment, but no guarantee). If we need guarantee, I can move this step to happen upon docker daemon is up and before restarting docker daemon after creating cbr0. This may slow down cluster creation a little bit.